tests/blackbox/src/common/config.py

```diff
@@ -39,7 +41,9 @@ def __init__(self) -> None:
         self.test_cluster_auth_token = get_env("TEST_CLUSTER_AUTH_TOKEN")
         self.aicore_deployment_id_gpt4 = get_env("AICORE_DEPLOYMENT_ID_GPT4")
```

Comment:
can this be cleaned up?

Reply:
Let us keep it here so we can rather easily switch between models, whenever we run the evaluation tests locally.

Reply:
Done.